### ISSUE TITLE
Empty `bcache` error

### ISSUE THREAD

**Issue by maisiliym** [lewis6991/gitsigns.nvim#32]
`gh` Lost the original issue body. So Li wont type it all out again. Heres the error message
```
Error executing vim.schedule lua callback: .../home-manager/start/gitsigns-nvim/lua/gitsigns/async.lua:21: ...r/pack/home-manager/start/gitsigns-nvim/lua/gitsigns.lua:594: attempt to index local 'bcache' (a nil value)
```

**Comment by lewis6991** [comment]
Are you able to reliably reproduce this? There's already a known issue of the cache disappearing but no one knows how to reproduce it.

I potentially think it could be a luajit problem but hopefully not.

**Comment by lewis6991** [comment]
Can you update and see if the issue is fixed (with 6433129ce42257e20e6518a0c73ee7636b5a8b3d), I found a bug that could cause this.
